### ISSUE TITLE
Update Household question on eligibility questionnaire

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#836]
In light of user feedback, we'd like to update the Household question on the eligibility questionnaire. The current question is 'How many people will be living in this property?'. The new questions should be 'How many people will live in your next rental, including yourself?'
